tools/cli/factory.go

```diff
@@ -22,29 +22,26 @@ package cli
 
 import (
 	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
 	"time"
 
-	"go.uber.org/yarpc/peer"
-	"go.uber.org/yarpc/peer/hostport"
-	"go.uber.org/yarpc/transport/grpc"
-	"go.uber.org/yarpc/transport/tchannel"
-	"google.golang.org/grpc/credentials"
-
 	"github.com/olivere/elastic"
+	adminv1 "github.com/uber/cadence-idl/go/proto/admin/v1"
+	apiv1 "github.com/uber/cadence-idl/go/proto/api/v1"
```

Comment:
this is kinda the worst-case scenario for not grouping -idl (and arguably go.uber.org/cadence) into the "project" imports.  or some "us"-group like goimports-reviser supports.

but on balance I think I prefer to cut a strict line between _literally this project_ and everything else, vs not.